DESCRIPTION

```diff
@@ -37,6 +42,7 @@ Imports:
     cli
 Suggests: 
     testthat (>= 2.1.0),
+    curl,
```

Comment:
I `git blame`'d to see where this came from and I added it in https://github.com/metrumresearchgroup/rbabylon/commit/37b9d6735dcc462dbf4ded420803b4694a7ac7ab
but I have no idea why. I'll take it out and see if anything breaks.

Reply:
well, it seems jsonlite uses curl under the hood so `devtools::check()` fails if it's not in Suggests. See this screenshot from https://github-drone.metrumrg.com/metrumresearchgroup/rbabylon/758/1/3

<img width="652" alt="Screen Shot 2020-06-25 at 11 35 11 AM" src="https://user-images.githubusercontent.com/7806083/85751280-6f04d400-b6d8-11ea-948e-ddb67c19faa1.png">

Reply:
One way to avoid the `curl` dependency would be to download the API response before passing to `fromJSON()`, for example
```r
tmp <- tempfile()
download.file("https://api.github.com/rate_limit", destfile = tmp, quiet = TRUE)
res <- jsonlite::fromJSON(tmp, simplifyDataFrame = FALSE)
```